frontend/src/pages/miscellaneous/ProfilePage.tsx

```diff
@@ -182,6 +192,51 @@ export default function ProfilePage() {
         window.location.reload();
     };
 
+    const handleImageChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
Similarly, handling of axios should not be here, should be handled under src/services/user-service. This function should be moved to the services/user-service. Then you call that function here.